query/security/wcpm/src/main/java/org/codice/ddf/admin/security/wcpm/persist/SaveContextPolices.java

```diff
@@ -113,6 +118,17 @@ public void validate() {
                 stsServiceProps));
     }
 
+    private void checkForTrailingSlashes(){
+        if(contextPolicies.getList()
+                .stream()
+                .map(ContextPolicyBin::contexts)
+                .flatMap(Collection::stream)
+                .anyMatch(p -> TRAILING_SLASH_PATTERN.matcher(p)
+                        .matches())) {
+            addErrorMessage(invalidPathTrailingSlash(contextPolicies.path()));
```

Comment:
Since we're not doing the validation in `ContextPath.java`, the path given to the `invalidPathTrailingSlash` error is not the path of the context path that failed but the context policy's path. Will that be an issue?